src/control/cmd/dmg/telemetry.go

```diff
@@ -0,0 +1,334 @@
+//
+// (C) Copyright 2019-2021 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/user"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/daos-stack/daos/src/control/common"
+)
+
+type telemCmd struct {
+	Configure telemConfigCmd `command:"config" description:"Configure telemetry"`
+	Run       telemRunCmd    `command:"run" description:"Launch telemetry system"`
+}
+
+type telemConfigCmd struct {
+	logCmd
+	cfgCmd
+	jsonOutputCmd
+	InstallDir string `long:"install-dir" short:"i" description:"Install directory for telemetry binary"`
```

Comment:
@kjacque expressed a desire to have some control over where the binary is installed. It's not required though. We can't assume that prometheus could be installed alongside all of the other binaries produced by a DAOS build because dmg may be running on a node where none of that stuff exists.